### PR TITLE
[Feature #160577623] Serves the article read time functionality

### PR DESCRIPTION
**Pull request by SrMoffat**
## What does this PR do?
Serves the article read time functionality `api/articles` endpoints 

## Pivotal Tracker Story
Users should be able to see time it takes to read an article.

## Tasks accomplished 

| Endpoint        | Functionality        | Branch  |Pivotal Tracker
| ------------- |-------------| ----------| -------------|
| api/articles/| View an article |      `ft-read-time-160577623`      |  Story is [here](https://www.pivotaltracker.com/n/projects/2198535/stories/160577623) |

## Manual Testing
`Run server` --> **python manage.py runserver**
`Postman` 
1. Create Article
`POST` on `http://127.0.0.1:8000/api/articles/` with:
``` 
{
   "article":
                {
		"title": "Article Readtime PR",
		"description": "The article readtime shows how long it takes to read",
		"body": "The article readtime is calcilated using TRT = NoW/275 + [(10/60)*NoI)"
		}
}
```
2. Get the Articles
When you get the articles using:
`GET` on `http://127.0.0.1:8000/api/articles/`
You expect to see the readtime of the article:
screen shot 2018-10-12 at 14 50 02

### CODE REVIEW

**Comment by mutaimwiti** [comment]
@SrMoffat kindly consider renaming the PR to conform to our convections.
@mutaimwiti kindly be more specific. As at now I feel like am well within the conventions. Thanks

**Review by gitaumoses4 — COMMENTED** [review]
@SrMoffat I see you have commits that do not belong to this branch.
Did you rebase onto develop before creating this PR?
@gitaumoses4 I had but it was a outdated version of develop. I have now rebased the latest develop. Is that why your commits are all over the PR?

File: authors/apps/articles/serializers.py [hunk 1]
```diff
@@ -81,14 +83,35 @@ class Meta:
...
+        NoW = Number of Words (in the article)
+        NoI = Number of Images (in the article)
+        """
+        image = obj.image
+        words = obj.body.split()
+        readtime = (len(words)/275.0)*60
+        image_readtime = 10  # Currently we allow a single image (10s)
+
+        if image == "":
+            print(readtime)
```
Comment (esirK) [inline authors/apps/articles/serializers.py]: This debug statements should be removed after testing. Even better, you can use `ipdb` for debugging.

**Comment by esirK** [comment]
add tests and work on comments left. Otherwise, it looks good.

File: authors/apps/articles/serializers.py (hunk 1 above)
Comment (gitaumoses4) [reply authors/apps/articles/serializers.py]: I'm glad I saw this. Thanks for the tip on `ipdb` @esirK

Comment (SrMoffat) [reply authors/apps/articles/serializers.py]: Yeah, I mistakenly pushed the print statements, I prefer using `ipdb` when facing a serious debugging issue or semantic errors. Here I just wanted to see how the data is flowing within the scope of the function before I return it `readtime`. Thanks for this, I have fixed it.

**Comment by esirK** [comment]
Looks good but remove all commits un-related to the task.

**Review by charisschomba — APPROVED** [review]

**Pull request merged**